documentation/endpoints/reactions.md

```diff
@@ -1,5 +1,84 @@
 ## Reactions
 
+Retrieve all Reaction data 
+
+```
+GET /api/v2/reactions
+```
+
+Example Response:
+
+```
+{
+  "data": [
+    {
+      "reaction_id": "1",
+      "northstar_id": "1234",
+      "postable_id": "4",
+      "postable_type": "Rogue\\Models\\Photo",
+      "created_at": {
+        "date": "2016-10-12 17:31:41.000000",
```

Comment:
@katiecrane ah thank you for catching that! I forgot to format with `toIso8601String()` in the transformer - updated!